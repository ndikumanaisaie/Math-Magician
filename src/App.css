.wrapper {
  width: 440px;
  height: 540px;
  padding: 10px;
  margin: auto;
  border-radius: 10px;
  background-color: #485461;
  background-image: linear-gradient(315deg, #485461 0%, #28313b 74%);
}

.screen {
  height: 100px;
  width: 100%;
  margin-bottom: 10px;
  padding: 0 10px;
  background-color: #4357692d;
  border-radius: 10px;
  display: flex;
  align-items: center;
  justify-content: flex-end;
  color: white;
  font-weight: bold;
  box-sizing: border-box;
}

.button-box {
  width: 100%;
  height: calc(100% - 110px);
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-template-rows: repeat(5, 1fr);
  grid-gap: 2px;
}

.sign-btn {
  background-color: #fbb034;
  border: none;
  font-size: 24px;
  font-weight: bold;
  cursor: pointer;
  outline: none;
}

.sign-btn:hover {
  background-color: rgb(255, 221, 0);
}

.keypad {
  border: none;
  background-color: gray;
  font-size: 24px;
  font-weight: bold;
  cursor: pointer;
  outline: none;
}

.keypad:hover {
  background-color: rgb(211, 210, 217);
}

.zero {
  grid-column: 1 / 3;
  background-color: gray;
  border: none;
  font-size: 24px;
  font-weight: bold;
  cursor: pointer;
  outline: none;
}

.zero:hover {
  background-color: rgb(211, 210, 217);
}
