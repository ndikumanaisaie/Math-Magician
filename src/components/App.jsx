import React from 'react';
import Calculator from './Calculator';

const App = () => (
  <div>
    <Calculator />
  </div>
);

export default App;
