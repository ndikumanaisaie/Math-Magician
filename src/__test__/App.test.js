import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

import App from '../components/App';

it('renders correctly App component', () => {
  const app = renderer.create(
    <MemoryRouter>
      <App />
    </MemoryRouter>,

  ).toJSON();
  expect(app).toMatchSnapshot();
});
