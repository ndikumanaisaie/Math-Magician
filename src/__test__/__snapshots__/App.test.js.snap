// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly App component 1`] = `
<div
  className="main-container"
>
  <div
    className="nav-bar"
  >
    <h2>
      Math Magician
    </h2>
    <div
      className="nav"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <span
        className="vertical-bar"
      />
      <a
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      <span
        className="vertical-bar"
      />
      <a
        href="/quotes"
        onClick={[Function]}
      >
        Quotes
      </a>
    </div>
  </div>
  <section
    className="home-section"
  >
    <h2
      className="home-title"
    >
      Welcome to Math Magician
    </h2>
    <div
      className="paragraph"
    >
      <p>
        This calculator will help you make basic math calculation. It is build to make it easy for all fun of maths to do their calculation easily.
      </p>
      <p>
        This application will give chance to student and small business owner do calculations. For student, it will help them do math that is complicated to do by themselves otherwise it is not a good idea for student to use calculators it prevents developing thinking skills
      </p>
    </div>
  </section>
</div>
`;
